### PR TITLE
update to 2.2.0

### PR DESCRIPTION
**Pull request by mcg1969**
- Updated version and sha256
- added wheel dependency
- removed noarch
- made importlib_metadata dependency specific to py37 and earlier (per source code)
- added constraints to setuptools and setuptools_scm (per source code)
- cleaned up metadata
- added pip check

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,33 +12,39 @@ source:
...
 
 requirements:
   host:
     - pip
+    - wheel
     - python
-    - setuptools
-    - setuptools_scm
+    - setuptools >=42
+    - setuptools_scm >=3.4.1
```
Comment (mcg1969) [inline recipe/meta.yaml:26]: reference: https://github.com/jsonpickle/jsonpickle/blob/main/pyproject.toml#L2

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,33 +12,39 @@ source:
...
     - toml
   run:
     - python
-    - importlib_metadata
+    - importlib_metadata  # [py<38]
```
Comment (mcg1969) [inline recipe/meta.yaml:30]: reference: https://github.com/jsonpickle/jsonpickle/blob/main/setup.cfg#L46-L47

File: recipe/meta.yaml [hunk 3]
```diff
@@ -12,33 +12,39 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  noarch: python
+  # noarch: python
+  skip: true  # [py<37]
```
Comment (mcg1969) [inline recipe/meta.yaml:16]: reference: https://github.com/jsonpickle/jsonpickle/blob/main/setup.cfg#L45

**Comment by mcg1969** [comment]
Transient osx-64 failure. Packages built successfully but failed in the upload stage

File: recipe/meta.yaml [hunk 4]
```diff
@@ -12,33 +12,43 @@ source:
...
+    - pip
   imports:
     - jsonpickle
+  commands:
+    - pip check
 
 about:
-  summary: Python library for serializing any arbitrary object graph into JSON. It can take almost any Python object and turn the object into JSON. Additionally,
-    it can reconstitute the object back into Python.
   home: http://jsonpickle.github.io/
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: Could you please fix home url with HTTPS?

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml (hunk 4 above)
Comment (mcg1969) [reply recipe/meta.yaml]: yes, will do!

**Comment by skupr-anaconda** [comment]
An error on `osx-64`:
`prefect.exceptions.ClientError: [{'path': ['secret_value'], 'message': 'An unknown error occurred.', 'extensions': {'code': 'INTERNAL_SERVER_ERROR'}}]`

Something is wrong with Prefect

**Comment by mcg1969** [comment]
@skupr-anaconda yep, I've reported this to team sirius

**Review by cbouss — APPROVED** [review]
LGTM

**Pull request merged**